doc/7-systems/virtual-boy.md

```diff
@@ -51,5 +51,9 @@ this chip uses the [Virtual Boy](../4-instrument/virtual-boy.md) instrument edit
 the following options are available in the Chip Manager window:
 
 - **Waveform storage mode**: selects how will waveforms be loaded.
-  - Dynamic: allows switching between many waves, but it is not confirmed whether this really works on hardware.
   - Static: loads the first 5 waves only.
+  - Dynamic: allows switching between more than 5 waves, but loading a new wave forces all channels off and back on again, usually with an audible click.
+
+- **Disable anti-phase-reset**: normally, changing a playing note's waveform resets its phase, causing a click. this eliminates it.  _note:_ this is not hardware accurate!
```

Comment:
This option does not involve hardware accuracy. The "I don't care" one does though.